src/LocalOrchestrationClient.php

```
@@ -0,0 +1,169 @@
+        $orchestrationId = OrchestrationId($name, $id ?? Uuid::uuid4()->toString());
+        $stateId = StateId::fromOrchestrationId($orchestrationId);
+
+        $event = WithOrchestration::forInstance(
+            $stateId,
+            StartOrchestration::fromArray(SerializedArray::fromArray($args)),
+        );
+
+        $eventDescription = new EventDescription($event);
+        $userArray = $this->userContext ? Serializer::serialize($this->userContext) : null;
+
+        $sequence = emit_event($userArray, $eventDescription->toArray(), $stateId->id);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sequence'. (Unused Code Rules)

(UnusedLocalVariable)